### PR TITLE
Use openRC depend rather than executing "service docker start" from "workflow-helper" script / Log workflow-helper output into the file

### LINKED ISSUE #24: Use openRC depend rather than executing "service docker start" from "workflow-helper" script
https://github.com/OpenRC/openrc/blob/master/service-script-guide.md#the-depend-function

### LINKED ISSUE #26: Log workflow-helper output into the file
So it can be checked after the execution for investigating.

### PR DESCRIPTION
**Pull request by RafaelAnsay**
Fixed #24 
Also Fixed #26 
- [ ] Changelog updated

### CODE REVIEW

**Comment by RafaelAnsay** [comment]
depend() {need docker} in workflow-helper-rc dynamically starts docker

File: installer/workflow-helper.sh [hunk 1]
```diff
@@ -42,6 +42,8 @@ wget "$packet_base_url/ca.pem"
 mkdir -p /etc/docker/ /etc/docker/certs.d/ "/etc/docker/certs.d/$docker_registry"
 cp ca.pem "/etc/docker/certs.d/$docker_registry/ca.crt"
 
+# wait for docker to fully start
+sleep 3
```
Comment (invidian) [inline installer/workflow-helper.sh]: There is `until` loop right after it, which will retry if command fails. Is this sleep really needed?

**Review by invidian — CHANGES_REQUESTED** [review]
Nice PR @RafaelAnsay. I'd squash first 2 commits, to avoid bloating git history. Other than that it looks good to me!

File: installer/workflow-helper.sh (hunk 1 above)
Comment (RafaelAnsay) [reply installer/workflow-helper.sh]: yes. in my dev environment docker login fails at least once because docker has not fully started.

Comment (invidian) [reply installer/workflow-helper.sh]: But if there is a `until` loop, it should retry, so I think it's fine if it fails?

Comment (RafaelAnsay) [reply installer/workflow-helper.sh]: sure i will remove the 1st sleep i added

**Comment by parauliya** [comment]
Hey @RafaelAnsay ,
Have you tested this with the end to end workflow? If not please test this and post the result here.

**Comment by RafaelAnsay** [comment]
sure i will post the results here

**Comment by parauliya** [comment]
> sure i will post the results here

Hi @RafaelAnsay , what is the status of this PR? I think it has been in an open state for quite some time. Please provide the test results so that we can merge this PR asap.

**Pull request closed without merging**